lib/tasks/technical_metadata.rake

```diff
@@ -24,6 +24,7 @@ namespace :techmd do
   task :generate_for_moab_list, %i[force] => :environment do |_task, args|
     force = args[:force] == 'true'
     File.foreach('druids.txt').map(&:strip).each_with_index do |druid, index|
+      druid = "druid:#{druid}" unless druid.start_with?('druid')
```

Comment:
allow the list of druids that are sent to **not** have the "druid:" prefix (since this is what you get when you export from argo)